src/javascript/JContent/ContentTree/ContentTree.jsx

```diff
@@ -25,7 +26,8 @@ export const ContentTree = ({lang, siteKey, path, openPaths, setPath, openPath,
         openableTypes: item.config.openableTypes,
         selectableTypes: item.config.selectableTypes,
         queryVariables: {language: lang},
-        hideRoot: item.config.hideRoot
+        hideRoot: item.config.hideRoot,
+        sortBy: SORT_CONTENT_TREE_BY_NAME_ASC
```

Comment:
Does this sort pages tree as well and if it does do we want to do this? We also need to have yarn.lock updated with updated version of dependencies.

Reply:
Yes it sorts the pages tree as well
I will check with @fpral on that one
Thanks (I will note it to update the yarn.lock with the new dependency) good catch

Reply:
Updated the package.json and yarn.lock
Waiting on feedback from @fpral

Reply:
Feed-back to update this portion to exclude for item.key === pages